incubator/synapse/templates/ingress.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.ingress.enabled -}} 
+{{- $fullName := include "synapse.fullname" . -}}
+{{- $servicePort := .Values.service.port -}} 
+{{- $ingressPath := .Values.ingress.path -}} 
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    app: {{ template "synapse.name" . }}
```

Comment:
please use the new labels. the latest helm does that.
or follow https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#names-and-labels

Reply:
Fixed